pages/content/amp-dev/page-experience.html

```diff
@@ -12,22 +13,37 @@
     <div class="ap-o-stage">
       <div class="ap--container-fluid">
         <div class="ap-o-stage-content">
-          <h1 class="ap-o-stage-content-headline">Web Vitals and AMP Headline</h1>
+          <h1 class="ap-o-stage-content-headline">
+            Optimize your AMP pages for Google's Page Experience ranking signal
+          </h1>
         </div>
         <div class="ap-o-stage-image">
           {% do doc.icons.useIcon('icons/vitals-header.svg') %}
           <svg>
-            <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#vitals-header"></use>
+            <use
+              xmlns:xlink="http://www.w3.org/1999/xlink"
+              xlink:href="#vitals-header"
+            ></use>
           </svg>
         </div>
       </div>
     </div>
   </section>
 
+  {% if podspec.env.name == 'production' %}
+  <section class="ap-t-pixi-intro ap-t-pixi-coming-soon">
+    <p class="ap-t-pixi-intro-text">
+      Coming soon
```

Comment:
suggestion, rephrase to: "Coming soon! Read more about how AMP helps achieve a great page experience in our <a href="https://blog.amp.dev/2020/05/28/amp-page-experience/"> blog post</a>."

Reply:
Done